packages/foreman/nodejs-theforeman-vendor/nodejs-theforeman-vendor.spec

```diff
@@ -39,6 +39,9 @@ cp -pfr scss %{buildroot}%{nodejs_sitelib}/%{npm_name}
 %doc readme.md
 
 %changelog
+* Fri Dec 13 2019 vagrant 3.8.0-1
```

Comment:
You are running inside vagrant box, aren't you? :wink:

Could you add your name and email? See the log entries below.